src/ResourceManager/DeploymentManager/Commands.DeploymentManager/Commands/RemoveServiceTopology.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.DeploymentManager.Commands
+{
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.DeploymentManager.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DeploymentManagerServiceTopology", SupportsShouldProcess = true), OutputType(typeof(bool))]
```

Comment:
This cmdlet needs to accept ResourceId and inputObject for piping scenarios. Look at [required parameter sets](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#required-parameter-sets).